Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG GO_VERSION=1.17
+
+FROM golang:${GO_VERSION}-alpine as build
+
+RUN apk --no-cache add make ca-certificates 
+WORKDIR /src
+COPY go.mod go.sum /src/
+RUN go mod download
+COPY . /src/
+RUN make build
+ENTRYPOINT [ "/src/bin/in-toto" ]
+
+FROM debian:buster
```

Comment:
I suggest we just use Google's distroless containers here. They have a smaller attack surface and provide ca certificates as well :)

`FROM gcr.io/distroless/base`

Reply:
I agree that is better. Changed.